benefits/core/migrations/0002_sample_data.py

```diff
@@ -74,6 +92,7 @@ def load_sample_data(app, *args, **kwargs):
 -----END CERTIFICATE-----
 """,
         label="Dummy certificate",
+        remote_loaded=True,
```

Comment:
`remote_loaded=True` is a minor shortcut for these `PemData` that already contain their `text`